app-configuration-hubs/create-group-and-resource.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash

```

Comment:
The files have DOS line endings, which seems to be causing some problems with the bash scripts running on UNIX platforms. Make sure that they're CR instead of CRLF, and properly UTF-8.

Reply:
Line endings updated.